ol3-cesium-plugin/garastersynchronizer.js

```diff
@@ -0,0 +1,81 @@
+goog.provide('ga.GaRasterSynchronizer');
+goog.require('olcs.RasterSynchronizer');
+
+
+
+/**
+ * Handle mapping OL3 2D layers to Cesium 3D Imageries in EPSG:4326.
+ * @param {!ol.Map} map
+ * @param {!Cesium.Scene} scene
+ * @constructor
+ * @extends {olcs.RasterSynchronizer}
+ * @api
+ */
+ga.GaRasterSynchronizer = function(map, scene) {
+  goog.base(this, map, scene);
+};
+goog.inherits(ga.GaRasterSynchronizer, olcs.RasterSynchronizer);
+
+
+/**
+ * @override
+ */
+ga.GaRasterSynchronizer.prototype.convertLayerToCesiumImagery =
+    function(olLayer, viewProj) {
+
+  /**
+   * @type {Cesium.ImageryProvider}
+   */
+  var provider = null;
+  var source = olLayer.getSource();
+
+  if (source instanceof ol.source.Vector) {
+    return null;
+  }
+
+  // Read custom, non standard properties
+  var layerId = olLayer['id'];
+  var layerTime = olLayer['time'];
+
+  if (source instanceof ol.source.WMTS) {
+    provider = new Cesium.WebMapTileServiceImageryProvider({
+      url: 'https://mf-chsdi3.dev.bgdi.ch/mom_fix_1594/1.0.0/' +
+         layerId + '/{Style}/' + layerTime +
+         '/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png',
+      layer : layerId,
+      style : 'default',
+      format : 'image/png',
+      tileMatrixSetID : '4326',
+      // Is a custom Cesium.GeographicTilingScheme necessary?
+      tilingScheme: new Cesium.GeographicTilingScheme({
+        numberOfLevelZeroTilesX: 2,
+        numberOfLevelZeroTilesY: 1,
+        rectangle: new Cesium.Rectangle(
+          Cesium.Math.toRadians(-180.0),
+          Cesium.Math.toRadians(-90.0),
+          Cesium.Math.toRadians(180.0),
+          Cesium.Math.toRadians(90.0))
+      }),
+      maximumLevel: 19
```

Comment:
This is probably layer dependant (minresolution, maxresolution from the layersConfig). Not sure how it could be translated to 4326 tilematrix set. @procrastinatio should know.